packages/core-utils/package.json

```diff
@@ -44,10 +44,12 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
+    "@ethersproject/abi": "^5",
```

Comment:
Same comment as above with respect to adding to `devDependencies` instead of `dependencies`. Here you will need to add the following:

- `@ethersproject/abi`
- `@ethersproject/abstract-signer`
- `@nomiclabs/ethereumjs-vm`